@xen-orchestra/backups/writers/_MixinBackupWriter.js

```diff
@@ -14,22 +14,26 @@ exports.MixinBackupWriter = (BaseClass = Object) =>
       this._lock = undefined
     }
 
+    async _cleanVm(options) {
+      try {
+        await this._adapter.cleanVm(getVmBackupDir(this._backup.vm.uuid), { ...options, onLog: warn })
+      } catch (error) {
+        if (error?.code !== 'ENOENT') {
```

Comment:
Why is this necessary?

Reply:
In case of non-existent sub-folders due to a non-existent backups.

Reply:
I believe `cleanVm` can only throw this error if the passed dir does not exist, which is no longer possible.

IMHO, `cleanVm` should not throw in case of other problems, simply warn.